addons/account/wizard/accrued_orders.py

```diff
@@ -167,7 +167,7 @@ def _ellipsis(string, size):
                     o.order_line.with_context(accrual_entry_date=self.date)._compute_untaxed_amount_invoiced()
                     o.order_line.with_context(accrual_entry_date=self.date)._compute_qty_to_invoice()
                 lines = o.order_line.filtered(
```

Comment:
We definitely have to add a comment to explain what we're doing here
Why `is_purchase or not l.is_downpayment` ? Please explain it in a comment